daprdocs/content/en/operations/resiliency/policies.md

```diff
@@ -91,3 +93,41 @@ spec:
+
+Below is a table that describes Dapr's default retries and the policy keywords to override them: 
+
+| Capability             | Override Keyword                 | Default Retry Behavior                                                                                                                                                               | Description                                                                                                                           |
+| ------------------     | -------------------------        | ------------------------------                                                                                                                                                       | -----------------------------------------------------------------------------------------------------------                           |
+| Service Invocation     | DaprBuiltInServiceRetries        | Per call retries are performed with a backoff interval of 1 second up to a threshold of 3 times.                                                                                     | Sidecar-to-sidecar requets (via a service invocation call) that fails and results in a gRPC code `Unavailable` or `Unauthenticated`   | 
+| Actors                 | DaprBuiltInActorRetries          | Per call retries are performed with a backoff interval of 1 second up to a threshold of 3 times.                                                                                     | Sidecar-to-sidecar requests (to a remote actor) that fails and results in a gRPC code `Unavailable` or `Unauthenticated`              |
+| Actor Reminders        | DaprBuiltInActorReminderRetries  | Per call retries are performed with an exponential backoff wtih an initial interval of 500ms and up to a maximum of 60s for a duration of 15m                                        | Requests that fail to persist an actor reminder to a state store                                                                      | 
+| Initialization Retries | DaprBuiltInInitializationRetries | Per call retries are performed 3 times wtih an exponential backoff with an initial interval of 500ms and for a duration of 10s                                                       | Failures when making a request to an application to retrieve a given spec. For example, failure to retrieve a Subscription spec       | 
+
+
+Example of overriding default retries for Service Invocation:
+```yaml
+spec:
+  policies:
+    # Retries are named templates for retry configurations and are instantiated for life of the operation.
+    retries:
+      DaprBuiltInServiceRetries: # Overrides default retry behaivor for service-to-service calls
+        policy: constant
+        duration: 5s
+        maxRetries: 10
+
+      retryForever: # A custom retry policy will replace default retries and the target will rely soley on the named policy. 
```

Comment:
Is this just showing another retry policy? I find the comment on customer retry policy confusing here. What is custom about this and this has nothing to do with the default policies. Also default and BuiltIn are two words for the same thing and we should just pick one.

Reply:
When you define a retry policy and assign it to a component, for example a target service. That target will always use when you've defined instead of any dapr default retries. 

Another way to say this is -  it's impossible to have default retries and custom retries stacked. You can only have one or the other at any time